examples/is-like/is_like.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina/io;
```

Comment:
We can't really have an example for the `is-like` check right? That's only an internal implementation detail. BBEs focus on explaining the usage of concrete language features. This instead should've been 2 examples on convert() and stamp() functions IMO.

Reply:
+1 to have two samples. We can include them under 'types' section.

Reply:
Type-guard, type-convert examples are added. Mohan will add type-stamp